### PR TITLE
Reenable and refactor CSP

### PR DESCRIPTION
**Pull request by jeppester**
It was disabled by accident by `rails app:update`.

Refactor it to make it easier to group additions to the csp,
preventing a mess of added hosts

Co-authored-by: Thies Pierdola <thiespierdola@gmail.com>

### CODE REVIEW

**Comment by runephilosof-abtion** [comment]
https://github.com/abtion/epos-atc-headset-configurator/blob/main/config/initializers/content_security_policy.rb

**Comment by runephilosof-abtion** [comment]
Finally finished with pushing commits. Ready for review @jeppester

File: config/initializers/content_security_policy.rb [hunk 1]
```diff
@@ -6,28 +6,50 @@
...
+  form_action: [:self],
+  frame_ancestors: [:none],
+  connect_src: [:self],
+  default_src: [:self],
+  font_src: [:self],
+  img_src: [:self],
+  object_src: [:none],
+  script_src: [:self],
+  style_src: [:self]
+}.freeze
```
Comment (jeppester) [inline config/initializers/content_security_policy.rb:20]: What is the purpose of this layer of indirection?

Comment (runephilosof-abtion) [reply config/initializers/content_security_policy.rb:20]: See https://app.asana.com/0/1142794766483633/1208315235207752

Comment (runephilosof-abtion) [reply config/initializers/content_security_policy.rb:20]: Maybe it deserves a comment

Comment (runephilosof-abtion) [reply config/initializers/content_security_policy.rb:20]: It makes it possible to add hosts to the policies more structured in commented groups, like the Rollbar addition on lines 23-25

Comment (jeppester) [reply config/initializers/content_security_policy.rb:20]: That makes total sense :+1: 

For some reason I did not notice that even though I looked through the code multiple times.

**Comment by jeppester** [comment]
I cannot approve this PR because I was the original author, but you can consider it approved :smile:

**Comment by jeppester** [comment]
I'm merging this

**Pull request merged**